pixiedust_node/npm.py

```diff
@@ -1,22 +1,35 @@
 
 import subprocess
+import os
+from .nodestdreader import NodeStdReader
+from pixiedust.utils.environment import Environment
 
 # npm helper
 # allows npm modules to be installed, removed and listed
 class Npm:
 
     # run an npm command
     def cmd(self, command, module):
+        # create node_modules
+        home = Environment.pixiedustHome
+        node_modules = os.path.join(home,'node_modules')
+        if not os.path.exists(node_modules):
+            os.makedirs(node_modules)
```

Comment:
@glynnbird hum... I thought npm would create node_modules automatically if not there. I guess it doesn't hurt to create it manually